app/assets/scripts/components/home/index.js

```diff
@@ -12,105 +21,369 @@ import {
+                  <dt>Indicators</dt>
+                  <dd><Link to='/indicators' title='Learn about the indicators'>20</Link></dd>
+                  <dt>Stories</dt>
+                  <dd><Link to='' title='Dive into the stories'>04</Link></dd>
+                </IntroStats>
+                <IntroStories>
+                  <IntroStoriesTitle>Did you know?</IntroStoriesTitle>
+                  <Story>
+                    <StoryTitle>{currentStory.title}</StoryTitle>
+                    <StoryProse>
+                      <p>{currentStory.prose}</p>
+                    </StoryProse>
+                    <StoryActions>
+                      <Button
+                        as='a'
+                        title='View the global map'
+                        to='/'
+                        variation='base-raised-light'
+                        useIcon={['chevron-right--small', 'after']}
+                      >
+                        Learn more
+                      </Button>
+                    </StoryActions>
+                  </Story>
+                </IntroStories>
+              </IntroCopy>
+              {/*
+              <IntroMedia>
+              </IntroMedia>
+              */}
+              <MbMap
+                ref={this.mbMapRef}
+                onAction={this.onMapAction}
+                layers={[]}
+                activeLayers={[]}
+                // date={this.state.timelineDate}
+                aoiState={null}
+                comparing={false}
+              />
+              <Previous
+                as='a'
+                title='Previous story'
+                disabled={storyIndex === 0}
+                to='/'
+                variation='base-raised-light'
+                useIcon={['chevron-left--small', 'after']}
+                hideText
+                onClick={this.prevStory}
               />
-              <IntroActions>
-                <Button
-                  as={CleanNavLink}
-                  size='large'
-                  title='View the global map'
-                  to='/getstarted'
-                  variation='primary-raised-dark'
-                >
-                  Getting Started
-                </Button>
-                <Button
-                  as={CleanNavLink}
-                  size='large'
-                  title='View the Indicators'
-                  to='/indicators'
-                  variation='primary-raised-dark'
-                >
-                  Indicators
-                </Button>
-                <Button
-                  as={CleanNavLink}
-                  size='large'
-                  title='View the Spotlight locations'
-                  to='/spotlight'
-                  variation='primary-raised-dark'
-                >
-                  Spotlight locations
-                </Button>
-              </IntroActions>
-            </PageConstrainer>
+              <Next
+                as='a'
```

Comment:
```suggestion
                element='a'
```
Because `Button` is a custom and complex component is better to use `element`.